database/migrations/20200916191917_users.js

```diff
@@ -1,15 +1,27 @@
 exports.up = async function (knex) {
-    await knex.schema
-    .createTable("users", (tbl) => {
-        tbl.increments("id").unsigned().primary()
-        tbl.text("username", 20).unique().notNullable()
-        tbl.text("password").notNullable();
-        tbl.text("phoneNumber").notNullable().unique()
-    })
+  await knex.schema.createTable("users", (tbl) => {
+    tbl.increments("id").unsigned().primary();
+    tbl.text("username", 20).unique().notNullable();
+    tbl.text("password").notNullable();
+    tbl.text("phoneNumber").notNullable().unique();
+  });
 
-}
+  await knex.schema.createTable("plants", (tbl) => {
+    tbl.increments().unsigned().primary();
+    tbl
+      .integer("user_id")
+      .references("id")
+      .inTable("users")
+      .onUpdate("CASCADE")
+      .onDelete("CASCADE");
+    tbl.text("nickname").notNullable();
+    tbl.text("species").notNullable();
+    tbl.integer("h20Frequency").notNullable();
+    tbl.text("image"); //img src/ URL for now
+  });
+};
```

Comment:
I do like the schema here and like the comment even, however the issue is that this is in the `users` migration file. I would like to either see this migration file renamed OR a new migration file added for `plants`